content/en/tests/setup/java.md

```diff
@@ -177,6 +183,9 @@ Set the following environment variables to configure the tracer:
 `DD_CIVISIBILITY_ENABLED=true` (Required)
 : Enables the Test Optimization product.
 
+`DD_TEST_SESSION_NAME`
+: Use this to identify a group of tests (for example: `unit-tests` or `integration-tests`).
+
 `DD_ENV` (Required)
```

Comment:
Should we remove `DD_ENV` then, or at least mark it as optional, to be consistent across languages?

Reply:
I'll remove the `(Required)` part but leave the rest, so the user knows the env var is there